bin/dev/legalscan.sh

```diff
@@ -1,4 +1,9 @@
 #!/usr/bin/env bash
+
+for PACKAGE in $(osc ls Cloud:Platform:buildpacks:dependencies); do
+  osc submitrequest --yes --message=legal-review Cloud:Platform:buildpacks:dependencies $PACKAGE Cloud:Platform:sources:buildpacks:reviewed
```

Comment:
Why move the dependencies to `Cloud:Platform:sources:buildpacks:reviewed` instead of `Cloud:Platform:buildpacks:dependencies:reviewed`? This would make more sense IMHO?

Reply:
I did not move them, we just also submit them there so they get review by legaldb.

Reply:
Is it much of an effort to rename the target repository?

Reply:
We need submit requests, which are only possible from one repo to another. So we need two anyway afaik.